backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java

```diff
@@ -18,16 +20,22 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
 
 class UploadServiceTest extends BaseServiceTest<UploadService> {
   public static final int PATIENT_PAGEOFFSET = 0;
   public static final int PATIENT_PAGESIZE = 1000;
 
   @Autowired private PersonService _ps;
+  @MockBean protected AddressValidationService _addressValidation;
+  private final StreetAddress address =
+      new StreetAddress("123 Main Street", null, "Washington", "DC", "20008", null);
 
   @BeforeEach
   void setupData() {
     initSampleData();
+    initSampleData();
```

Comment:
do you need this twice? :)

Reply:
Nope. I think this might be a rebase mistake